__tests__/api.password-reset.test.ts

```
@@ -0,0 +1,42 @@
+  });
+
+  test('environment variable controls site URL', () => {
+    const originalEnv = process.env.NEXT_PUBLIC_SITE_URL;
+    
+    // Test with custom site URL
+    process.env.NEXT_PUBLIC_SITE_URL = 'https://testero.ai';
+    const customURL = `${process.env.NEXT_PUBLIC_SITE_URL}/verify-email`;
+    expect(customURL).toBe('https://testero.ai/verify-email');
+    
+    // Test fallback
+    delete process.env.NEXT_PUBLIC_SITE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)